### PR TITLE
Feat/v2 docs

### PR DESCRIPTION
**Pull request by AndreasGassmann**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,34 +2,19 @@
...
-
-```
-npm i --save @airgap/beacon-sdk@2
-```
-
 ## Intro
 
 The `beacon-sdk` simplifies and abstracts the communication between dApps and wallets over different transport layers.
 
-The SDK is available on other platforms:
```
Comment (pascuin) [inline readme.md:29]: Keep this here

File: readme.md [hunk 2]
```diff
@@ -41,33 +26,71 @@ The documentation can be found [here](https://docs.walletbeacon.io/).
 npm i --save @airgap/beacon-sdk
 ```
 
-## Example
+## Upgrading from v1 to v2
```
Comment (pascuin) [inline readme.md:29]: create an entry for this in the docs as well

Comment (AndreasGassmann) [reply readme.md:34]: Done

File: readme.md (hunk 1 above)
Comment (AndreasGassmann) [reply readme.md:29]: I moved the SDKs back up here

**Pull request merged**